### DISCUSSION TITLE
2023-11-14

### DISCUSSION THREAD

**Discussion by ngsy** [10-rolling/rollingPolling#90] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] PostList 카드에 해당하는 폰트 적용
- [x] Editor 컴포넌트 입력값 버그 해결
- [x] list 페이지 완성
- [x] PostList Item 삭제 기능구현
- [x] 무한스크롤 기능 구현

### 📝 회의내용

- 오늘의 TMI

수영님 : 걱정했던 케이크 .. 눈 안그림 ㅠㅠ (쌍수전)
수님 : 어푸 안해줘서 서운하다, 감기 ㅠㅠ
윤혁님 : 가슴 운동함 ... 이것만 하나?
슬희님 : 23일 취소 ... 쏘리

- 현재 진행상황 공유

수영님 : 무한 스크롤
수님 : 삭제 기능, 수정사항 변경
윤혁님 : 반응형
슬희님 : list 페이지 기능 완성, 배포

- 수정이 필요한 부분 정리

카드 등록시 바로 카드가 안나타나는 문제
무한 스크롤 반응형때도 되는 건가?
배포주소로 들어갔을 때 페이지 url로 이동이 안되는 문제 - 슬희
api 실패했을 때 에러처리하기 (모달, 메세지 등) - 시간나면

- 할 일

충돌 해결하기
수정이 필요한 부분 처리하기

### 💪🏻 오늘 할일

- 개발하기
- **운동하기**
- 낮잠자기

### 🤔 특이사항

다들 어마어마 하시군요 역시 10팀..
감기조심하세요😖

**Comment by hanseulhee** [comment]
건강 또 건강 !!!!!

**Comment by oauch** [comment]
다들 아프지마요 😷
